ios/Podfile.lock

```diff
@@ -570,9 +570,9 @@ PODS:
     - React-Core
   - RNVectorIcons (10.0.0):
     - React-Core
-  - SDWebImage (5.18.2):
```

Comment:
Curious about what forced this change.

Reply:
I always have to delete podfile.lock, run  pod install to get ios working on my machine, pod install always does it. I revert it before pushing but looks like I've been missing it these days.